imod/protocols/protocol_fiducialModel.py

```diff
@@ -560,27 +496,21 @@ def xcorrStep(self, tsObjId):
 
         Plugin.runImod(self, 'tiltxcorr', argsTiltXCorr % paramsTiltXCorr)
 
-    def chopcontsStep(self, tsObjId):
-        '''
+    def chopcontsStep(self, tsId):
+        """
         $imodchopconts -StandardInput
         InputModel cryo_pt.fid
         OutputModel cryo.fid
         MinimumOverlap	4
         AssignSurfaces 1
-        '''
-
-        ts = self._getTiltSeries(tsObjId)
-        tsId = ts.getTsId()
-        extraPrefix = self._getExtraPath(tsId)
-
-        firstItem = ts.getFirstItem()
+        """
         MinimumOverlap = 4
         AssignSurfaces = 1
         LengthOfPieces = -1
 
         paramschopconts = {
-            'inputFile': os.path.join(extraPrefix, firstItem.parseFileName(suffix="_pt", extension=".fid")),
-            'outputFile': os.path.join(extraPrefix, firstItem.parseFileName(suffix="_gaps", extension=".fid")),
+            'inputFile': self.getExtraOutFile(tsId, suffix="pt", ext=".fid"),
+            'outputFile': self.getExtraOutFile(tsId, suffix="gaps", ext=".fid"),
```

Comment:
ext="fid" instead?

Reply:
Thanks, @azazellochg,

I've corrected those and made some more extension centralization.